packages/core/src/DataSchema.ts

```diff
@@ -157,6 +245,34 @@ export class DataSchema {
         return attestations.filter((attestation) => attestation !== null)
     }
 
+    /**
+     * Parse encoded schema, producing a dictionary of user-defined field names and attestation value
+     * @param data The raw data happends in UniRep contract.
```

Comment:
not sure what is meant here.
'The raw data appended to the Unirep contract.'  ?

Reply:
Because the data might be encoded
e.g.
```ts
{
    name: "data0",
    type: "uint8",
    value: 00000000
},
{
    name: "data1",
    type: 'uint8',
    value: 11111111
}
```
it can be encoded to 
| data1 |  data0 |
| :--: | :--: | 
| 11111111 | 00000000 |

We should submit the encoded data = `1111111100000000` to unirep smart contract
(instead of using two data fields)

on the other hand
if a data `1111111100000000` is submitted,
we should decode this data with this function. (to `data0` and `data1`)
So I call the unprocessed `1111111100000000` *raw data*

Reply:
ok! so yeah i think this change works:

> 'The raw data appended to the Unirep contract.'